src/webnn_native/dml/deps/src/device.h

```diff
@@ -0,0 +1,93 @@
+//-----------------------------------------------------------------------------

```

Comment:
Ditto, this could be a DEPS

Reply:
It has a similar reason. As mentioned in the README, the device wrapper code is based on the [Python bindings for DirectML samples](https://github.com/microsoft/DirectML/tree/master/Python). We proposed to make it more friendly for C++ usage. The DirectML is already a DEPS. So once the generic device wrapper is available in DirectML repo, we will use that and remove these files. Does it sound good? Also @wchao1115

Reply:
Sounds good.

Reply:
tracked by https://github.com/webmachinelearning/webnn-native/issues/3